clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -805,9 +793,12 @@ url, ignore, metalake, getOneRole(roles, CommandActions.REMOVE), name, privilege
     }
   }
 
-  private String getOneRole(String[] roles, String command) {
-    Preconditions.checkArgument(
-        roles.length == 1, command + " requires only one role, but multiple are currently passed.");
+  private String getOneRole(String[] roles) {
```

Comment:
Not 100% sure whether this is actually needed.
Sounds like this is not a new topic.

Reply:
Some commands support multiple roles some do not, so it is needed.